src/promptflow-devkit/promptflow/_sdk/_service/app.py

```diff
@@ -49,6 +49,10 @@ def heartbeat():
     return jsonify(response)
 
 
+def root():
+    return redirect(url_for("serve_trace_ui"))
```

Comment:
why need redict with a str here?  not directly call serve_trace_ui function?

Reply:
serve_trace_ui means the name of function?
![image](https://github.com/microsoft/promptflow/assets/26239730/44f133d1-caee-4673-8394-7e4ff7149bab)

Reply:
![image](https://github.com/microsoft/promptflow/assets/26239730/55f40418-b1fb-4d54-a028-b5f6157f2359)

Reply:
add test